pkg/plugin/plugin.go

```diff
@@ -498,10 +507,10 @@ func waitLinkUp(ovsDriver *ovsdb.OvsBridgeDriver, ofPortName string) error {
 				break
 			}
 		}
-		if i == linkstateCheckRetries {
+		if i == retries {
 			return fmt.Errorf("The OF port %s state is not up", ofPortName)
```

Comment:
Could you please improve this error message by suggesting to increase the number of retries/interval? So in case it fails on somebody's cluster, they would be able to triage it and fix it themselves.